### DISCUSSION TITLE
Endpoint for PU

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#98] [Q&A]
What is the endpoint IP for the PU ?
For RU it is 10.13.17.10

**Comment by ManuelB** [comment]
The ip is: 100.102.17.10

nslookup api.impfnachweis.info
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	api.impfnachweis.info
Address: 100.102.17.10

Feel free to base your solution on our open source solution: https://github.com/ere-health/dgc-ps-app (the solution is GPL, so looking into the code is allowed, copy and paste requires a commercial license)
